src/components/modal/modalHeader/modalHeader.tsx

```diff
@@ -7,15 +7,28 @@ const cx = classNames.bind(styles);
 
 interface ModalHeaderProps {
   title?: ReactNode;
+  headerContentClassName?: string;
   headerNode?: ReactNode;
+  headerDescription?: string;
   onClose: () => void;
 }
 
-export const ModalHeader: FC<ModalHeaderProps> = ({ title, onClose, headerNode }) => (
+export const ModalHeader: FC<ModalHeaderProps> = ({
+  title,
+  headerContentClassName = '',
+  headerDescription,
+  onClose,
+  headerNode,
+}) => (
   <div className={cx('modal-header')}>
-    <div className={cx('modal-header-content')}>
+    <div className={cx('modal-header-content', headerContentClassName)}>
       {title && <span className={cx('modal-title')}>{title}</span>}
       {headerNode && headerNode}
```

Comment:
Do we still need this prop?
Please ask UX team, as I don't see any usages of it within current UI service pages and don't any finished mockups where it can be used.